platforms-maturity-model/v1/index.md

```diff
@@ -497,19 +436,49 @@ Not only are standard frameworks leveraged, but there is an understanding that m
 * Over time the organization has collected data indicating a rise in build time of over 15%. This triggers negative developer experiences and once triggered, even if the build time is reduced below the original time, developers stay frustrated for longer. This insight drives the build team to set and adhere to a Service Level Objective (SLO), which enables early identification and improvement before instigating the negative cycle with their users.
 
 {{< /tab >}}
-{{< /tabs >}}
+{{< tab tabName="Context" >}}
 
-</div>
+<h4 style="color:gray;padding-bottom:4px;padding-top:12px"><i>Context of this work</i></h4>
 
-</br>
-</br>
+It's valuable to understand the context a document has been written in. The following sections lay out some context behind the model as well as some expectations for you, the reader.
 
----
+## Why provide a maturity model?
+
+This model encourages organizations to mature both their platform engineering discipline and the resulting platform through intentional planning. This effective planning and discipline is itself a precursor to effective platform evolution as organizational requirements change.
```

Comment:
IMO this is an interesting new section title but the content is not quite fit for purpose. If we want to tackle the decision to settle on a maturity model I think we need to ack the challenges these models have had in the past (consultant fodder) and why we still felt the structure provided the most possible value (narrowing focus, providing guideposts, acknowledging when behaviours result in less effective solutions etc).

I took a stab at it but am not happy with my version so decided it would be best to call this out and decide if we really want this section.

Reply:
Yeah, it felt like it might be a question folks ask but agree the paragraph is only a partial answer. Thinking about it.

Reply:
In the latest commit I moved this paragraph up into the top section and removed this heading entirely, see what you think.